drivers/dma/dma_stm32.c

```diff
@@ -11,14 +11,77 @@
  *        implemented in dma_stm32_v*.c
  */
 
+#include <soc.h>
+#include <init.h>
+#include <drivers/dma.h>
+#include <drivers/clock_control.h>
+#include <clock_control/stm32_clock_control.h>
+
```

Comment:
Some of these inclusions are redundant with below inclusions

Reply:
ok